code/clue.js

```diff
@@ -1,64 +1,297 @@
-// CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
-const mrGreen = {
-  firstName: "Jacob",
-  lastName: "Green",
-  color: "green",
-  description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
-  occupation: "Entrepreneur"
+const suspects = [
+  {
+    displayName: "Mr Green",
+    firstName: "Jacob",
+    lastName: "Green",
+    color: "#8DAB8D",
+    description: "He is a cat lover",
+    age: "32 years",
```

Comment:
This works, because you have made it to a string. But only for you to try different things: re write this as an `integer` (numbers only), and add `years` where you write this in the card

Reply:
a, but of course... thanks!